### ISSUE TITLE
--build-server-no-success-messages is ignored

### ISSUE THREAD

**Issue by imanushin** [Litee/SolutionCop#36]

**Comment by Litee** [comment]
I just checked v0.7.0 from command line and it seems to work - no success TeamCity message. End of console output is like this:

```
...
##teamcity[testIgnored name='SolutionCop.TreatWarningsAsErrors']
INFO: Analyzing projects using rule WarningLevel
##teamcity[testIgnored name='SolutionCop.WarningLevel']
##teamcity[testSuiteFinished name='SolutionCop']
INFO: Analysis finished!
INFO: No errors found!
```

Without flag it is:

```
...
##teamcity[testIgnored name='SolutionCop.TreatWarningsAsErrors']
INFO: Analyzing projects using rule WarningLevel
##teamcity[testIgnored name='SolutionCop.WarningLevel']
##teamcity[testSuiteFinished name='SolutionCop']
INFO: Analysis finished!
INFO: No errors found!
##teamcity[progressMessage text='PASSED - SolutionCop.xml']
```

**Comment by imanushin** [comment]
@Litee , what Team City thinks: 
1. First test arrived: stop displaying status messages (such as "Building project XXX", etc)
2. TeamCity shows only like "4 tests passed" (for 4 SC rules)
3. After Unit Test session start (e.g. after building) TeamCity starts to increment tests counter

What I think we should to improve:
1. All rules ignoration should be written to console without TeamCity mark
2. All passed tests info should be written to console without TeamCity mark
3. All errors are written with TeamCity mark

**Comment by Litee** [comment]
Sounds like one more strategy for console output. Let us do the following - I just have pushed re-factored command-lines parameters processing. You can raise PR with alternative strategy into ReportFormatType enum  + processing. IMO it will be faster ;)
